Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ghcr.io/void-linux/void-linux:20210312rc01-thin-x86_64-musl
```

Comment:
Might be worthwhile to add a build arg to override this at build time in case anyone ever needs to bisect something.